containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -465,26 +476,19 @@ class UnorderedMap {
     return is_insertable_map ? get_flag(erasable_idx) : false;
   }
 
-  bool begin_erase() {
+  bool begin_erase(const execution_space &space) {
```

Comment:
We still need the overload without execution space instance for backward-compatibility.

Reply:
Ok, I've written one. But I'm not sure it is done in the correct way. Could you check please?